src/hybrid_a_star_planner/main.py

```
@@ -1,6 +1,45 @@
+        obstacle_x,
+        obstacle_y,
+        AlgorithmConfig.XY_RESOLUTION,
+        AlgorithmConfig.YAW_RESOLUTION,
+    )
+
+    # 4. 运行 Hybrid A*
+    # 启用 plot_enabled=True 可以看到搜索过程中的中间路径
+    result = run_hybrid_a_star(start_state, goal_state, map_params, plot_enabled=False)
+
+    if result:
+        x_path, y_path, yaw_path, closed_set = result
```

Ruff (0.14.8)

Unpacked variable `closed_set` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)